doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -56,28 +56,27 @@ julia> 1234
 1234
 ```
 
-The default type for an integer literal depends on whether the target system has a 32-bit architecture
-or a 64-bit architecture:
+The default type for an integer literal depends on version of julia, 32bit or 64bit:
```

Comment:
hyphens?

Reply:
Thanks, hyphens had been added.

Reply:
sorry for troubling @garrison again, bob goes back with #29123 and if this is right way, i will do other pr(s) as well.